README.md

```diff
@@ -90,13 +93,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
 
 ## Libraries
 ### C
- - [mongo-c-driver](https://github.com/mongodb/mongo-c-driver) - Official C driver
+ - [mongo-c-driver](https://github.com/mongodb/mongo-c-driver) - Official MongoDB C driver
 
 ### C++
- - [mongo-cxx-driver](https://github.com/mongodb/mongo-cxx-driver) - Official C++ driver
+ - [mongo-cxx-driver](https://github.com/mongodb/mongo-cxx-driver) - Official MongoDB C++ driver
 
 ### C#/.NET ###
- - [mongo-csharp-driver](https://github.com/mongodb/mongo-csharp-driver) - Official C# driver
+ - [mongo-csharp-driver](https://github.com/mongodb/mongo-csharp-driver) - Official MongoDB C# driver
```

Comment:
I'm not a fan of these changes. I'm trying to stay as concise as possible for readability and mentioning "MongoDB" on a MongoDB list feels unneeded at best, so I usually try to avoid that. :)